<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background: #354152;
}

QLineEdit{
color:white;
border-radius:15px;
border: 1px solid;
border-color: #303030;
background: transparent;
padding-left: 15px;
font-size: 16px;
}

QPushButton{
border-radius: 15px;
	background-color: qlineargradient(spread:reflect, x1:1, y1:0.466409, x2:0.188, y2:0.482727, stop:0.823864 rgba(82, 185, 143, 255), stop:1 rgba(115, 185, 153, 255));
padding: 8px 16px;
color: white;
font-size: 20px;
}
QPushButton#btn_login:pressed{
color: black;
background: #008bd1

}

QCommandLinkButton{
font-size: 16px;
background: #4c5d75;

}

QCommandLinkButton:pressed{

	background-color: rgb(92, 186, 138);

}


QGroupBox{
border:none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>400</width>
        <height>531</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>311</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #7E8BA3;
font: 44px &quot;Helvetica Neue&quot;, sans-serif;
margin: 0px 0px 16px;</string>
       </property>
       <property name="text">
        <string>GİRİŞ SAYFASI</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_Sifre">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>330</y>
         <width>336</width>
         <height>42</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Şifre</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_login">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>430</y>
         <width>336</width>
         <height>42</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>GİRİŞ YAP</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_kullaniciAdi">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>336</width>
         <height>42</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Kullanıcı Adı</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>121</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/resimler/icons/damla.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="btn_kayit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>490</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>KAYIT OL</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_kontrol">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>390</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Admin Giriş</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>text_kullaniciAdi</tabstop>
  <tabstop>text_Sifre</tabstop>
  <tabstop>btn_login</tabstop>
  <tabstop>btn_kayit</tabstop>
 </tabstops>
 <resources>
  <include location="../ui_pages/resoruces.qrc"/>
 </resources>
 <connections/>
</ui>
